resharper/resharper-unity/src/UnityVersion.cs

```diff
@@ -288,5 +295,10 @@ public static Version GetVersionByAppPath(FileSystemPath appPath)
             });
             return version;
         }
+
+        public void InvalidateActualVersionForSolution()
```

Comment:
Might be better to call it `UpdateActualVersionForSolution()` - "invalidate" suggests clearing it, but not setting it again.